src/components/Canvas.vue

```diff
@@ -270,17 +291,46 @@ export default defineComponent({
     );
     const cursors = ref<Point[]>([]);
     const currentColor = ref<string>("");
+    const pivotPos = ref<Pos>({ x: 0, y: 0 });
     const moveToolPos = computed(() => {
       const { x, y } = cursors.value.reduce(
         ({ x, y }: Pos, cursor): Pos => {
           return {
-            x: x + cursor.x / cursors.value.length,
-            y: y + cursor.y / cursors.value.length,
+            x: Math.round(x + cursor.x / cursors.value.length),
+            y: Math.round(y + cursor.y / cursors.value.length),
           };
         },
         { x: 0, y: 0 }
       );
-      return { x: x - curw / 2, y: y - curw / 2 };
+      return { x, y };
+    });
+    const showToolHandles = computed(() => {
```

Comment:
Is this property used?

Reply:
Haha- not yet (T_T...I thought it will make too busy screen with too many tools. so at some point, I will switch the visibility of the tool. now we can omit it definitely! thanks!